mcs/build/start-compiler-server.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+if [ -x "$(command -v mono)" ] ; then
+    if test -f $1; then
+        echo . > /tmp/vbcs-log.txt
+        RoslynCommandLineLogFile=/tmp/vbcs-log.txt mono --gc-params=nursery-size=64m $1 -pipename:$2 &
```

Comment:
you should pass mono path from the build.make

Reply:
We could add it if it's the only missing

Reply:
How do I do that? I can do it then